threescale_api/resources.py

```diff
@@ -342,6 +342,19 @@ def __init__(self, *args, entity_name='api_doc', entity_collection='api_docs', *
     def url(self) -> str:
         return self.threescale_client.admin_api_url + '/active_docs'
 
+    def fetch(self, entity_id: int = None, **kwargs) -> dict:
+        """Fetch the entity dictionary
+        This is temporary overwrittening of default fetch method because of THREESCALE-5826
+        Args:
+            entity_id(int): Entity id
+            **kwargs: Optional args
+
+        Returns(dict): Resource dict from the 3scale
+        """
+        log.debug(self._log_message("[FETCH] Fetch ", entity_id=entity_id, args=kwargs))
+        params = {'id': entity_id}
+        return self.read_by(**params)
```

Comment:
this is kind of weird - `params` contains just entity id - I suppose you wanted to merge params with kwargs?

Reply:
you're right. I've missed it.